prow/config.yaml

```diff
@@ -643,10 +643,11 @@ presubmits:
     run_if_changed: 'gubernator'
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gubernator:0.2
-        env:
-        - name: GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
(saw https://github.com/kubernetes/test-infra/pull/3630 goes towards this, might as well make this explicit)